app/assets/stylesheets/modules/data_access.css.scss

```diff
@@ -192,31 +192,151 @@ section.data-access-next {
 
 .data-access-collections-list {
   list-style-type: disc;
-  padding-left: 2em;
+  margin-left: 2em;
+}
+
+.data-access-collections-list-panel {
```

Comment:
This is a question for @jsiarto and @mreese84 -- do you prefer longer class names or nested class names? These seem like long class names that could just be nested `.data-access-collections` on a surrounding element and `.name` `.body` `.state-text` kind of things nested?

Reply:
I do agree that the class name could be shortened possibly. I was mostly trying to follow the naming conventions I saw throughout. I've intentionally avoided nesting to keep specificity low. My BEM habits sneaking in, I suppose. I saw some similar things around the css files. Totally fine with whatever the group prefers though. I'm all ears, @jsiarto and @mreese84.

Reply:
Oddly enough, I don't have a strong opinion here. As long as the class names are semantic/descriptive, I'm good. In projects past we've gotten into some pretty serious specificity wars due to nesting. So while we aren't going full-on BEM here, I do generally support BEM, which would mean more classes and less nesting.

Reply:
We should keep nesting to a minimum--it's a nice feature, but can get out of hand quickly. I'm ok with this classname and setup.